tests/TestCollectors.py

```diff
@@ -37,8 +37,8 @@ def test_environment(self):
     def test_collector_metrics(self):
         metrics_yaml = yaml_read('tests/metrics.yaml')
         # every collector got to be tested in here
-        random_prometheus_port = random.randrange(9000, 9700, 1)
-        print("chosen testport: " + str(random_prometheus_port))
+        self.random_prometheus_port = random.randrange(9000, 9700, 1)
```

Comment:
what's the benefit of storing this in self?

Reply:
To globally increase the prometheus testing port after each run. Otherwise this will lead to an error while unregistering metrics.